include/logging/log_output.h

```diff
@@ -121,6 +127,18 @@ void log_output_msg_process(const struct log_output *output,
 void log_output_msg2_process(const struct log_output *log_output,
 			     struct log_msg2 *msg, uint32_t flags);
 
+/** @brief Process log messages v2 to SYS-T format.
+ *
+ * Function is using provided context with the buffer and output function to
+ * process formatted string and output the data in sys-t format
```

Comment:
This routine processes and outputs the supplied message in the SYS-T logging output format.